provisioning/miniprov/hare_mp/main.py

```diff
@@ -37,6 +37,12 @@
 from hare_mp.store import ConfStoreProvider
 from hare_mp.validator import Validator
 
+# Logger details
+haredeploy_logger_name = "hare-deployment-logger"
+haredeploy_log_file = "/var/log/seagate/hare/haredeployment/haredeployment.log"
+haredeploy_log_directory = "/var/log/seagate/hare/haredeployment/"
```

Comment:
`haredeployment` looks a bit strange to me. Does this name comes from the JIRA ticket?

Reply:
No I am referring this name to this directory.

Reply:
I know. But the folder name `haredeployment` looks a bit strange. If it is not a strict requirement, I would suggest other names like `"/var/log/seagate/hare/"` or `"/var/log/seagate/hare/deployment/"`.

It's obvious that if something lays under `seagate/hare`, then it relates to hare. So `hare` in `haredeployment` word sounds to be too much.

Reply:
Okay Sure. I'll give another name for it